src/write_to_vtk.cc

```diff
@@ -39,7 +41,7 @@
  */
 int writeToVtk(const Mesh& mesh,
 	       const std::vector<double> data,
-	       const std::string pfilename,
+	       fs::path filename,
```

Comment:
same as before. The filename is changed internally (this is part of the functionality), so const would not be correct